app/components-react/pages/RecordingHistory.tsx

```diff
@@ -53,6 +60,11 @@ class RecordingHistoryController {
 
   get uploadOptions() {
     const opts = [
+      {
+        label: `${$t('AI Highlighter')} ${$t('For Fortnite streams (Beta)')}`,
```

Comment:
im not sure i understand why these strings are separated?

Reply:
Thought it would make sense since both strings already existed on its own and i wanted to prevent duplicates. Merged to one string.

Reply:
got it, yea like i mentioned before sentence structure might change for different languages to this is another place where we want to avoid concatting translations, even if it causes some duplication